templates/Search/results.html.twig

```diff
@@ -1,4 +1,4 @@
-{% set isImperial = units.isImperial(app.request.locale) %}
+{% set isImperial = units.isImperial(app.request.locale, app.user) %}
```

Comment:
To be removed when dependency injection is fixed

Reply:
Same